vtex-apps/store-components
# Feat/ga4 send search event

#### What problem is this solving?

Send the new `search` event for the GTM app

#### How to test it?

* Go to this [branch](https://github.com/vtex-apps/store-components/tree/feat/ga4-send-serch-event)
* Link store-components to workspace
* Switch to [branch with new GTM version events](https://github.com/vtex-apps/google-tag-manager/tree/feat/settings-merge-ua-events) and link on workspace
* Search for any product on search bar on top
* Check on `window.dataLayer` if the item has the correct attributes


[Workspace](https://natandevuamerge--storecomponents.myvtex.com/)

#### Screenshots or example usage:
![Captura de Tela 2023-04-11 às 17 34 55](https://user-images.githubusercontent.com/36740164/231282189-3aebfa12-ee58-4710-a31d-7352cf4ce227.png)


#### Describe alternatives you've considered, if any.



#### Related to / Depends on



#### How does this PR make you feel? [:link:](http://giphy.com/)



![](https://media.giphy.com/media/26n6WywJyh39n1pBu/giphy.gif)
